boards/arm64/imx8mm_evk/imx8mm_evk.dts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx8mm_a53.dtsi>
+
+/ {
+	model = "NXP i.MX8MM A53";
+	compatible = "fsl,imx8mm";
+
+	aliases {
+		uart-2 = &uart2;
+		uart-4 = &uart4;
+	};
+
+	psci {
+		compatible = "arm,psci-0.2";
+		method = "smc";
+		label = "PSCI";
+	};
+
+	sram0: memory@40000000 {
+		reg = <0x93c00000 DT_SIZE_M(1)>;
+	};
+
+};
+
+&uart2 {
+	status = "okay";
+	current-speed = <115200>;
+};
+
+&uart4 {
```

Comment:
the `uart4` definition here should not be required, if you are not overriding properties in the board DTS